website/docs/r/azurerm_server_vulnerability_assessment.html.markdown

```diff
@@ -0,0 +1,111 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_server_vulnerability_assessment"
+description: |-
+  Adds Azure Vulnerability Assessment (Qualys) to a VM or Azure ARC server.
+---
+
+# azurerm_server_vulnerability_assessment
+
+Adds Azure Server Vulnerability Assessment (Qualys) to a VM.
+
+-> **NOTE** Azure Defender has to be enabled on the subscription in order for this resource to work.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resource-group"
+  location = "West US"
```

Comment:
I assume here should be "West Europe"

Reply:
Any reason you think we should use Europe instead of US?

Reply:
I assume it should be the the code style of terraform. You can reference other [resources](https://raw.githubusercontent.com/terraform-providers/terraform-provider-azurerm/master/website/docs/r/stack_hci_cluster.html.markdown) as sample.

Reply:
There is a wide range of locations are used in the documentation, West US is used in 123 other files. So keeping it for now.

Reply:
this is fine, thou fwiw we do typically default to west europe